qiskit_experiments/calibration_management/base_calibration_experiment.py

```diff
@@ -356,20 +356,11 @@ def get_schedule(
 
         return schedules
 
-    def circuits(self) -> List[QuantumCircuit]:
-        """A wrapper to introduce an optional hook to add circuit metadata."""
-        circuits = super().circuits()
+    def _add_cal_metadata(self, experiment_data: ExperimentData):
```

Comment:
Is there any reason to have this over the `_additional_metadata` function in BaseExperiment which can already add arbitrary dict values to ExperimentData.metadata?

Reply:
I think we can use `_additional_metadata`. I'll open a small PR to adjust to this.